tests/migration_test.go

```diff
@@ -109,6 +109,40 @@ var _ = Describe("Migrations", func() {
 		By("Verifying the VMI's is in the running state")
 		Expect(vmi.Status.Phase).To(Equal(v1.Running))
 	}
+	confirmVMIPostMigrationAborted := func(vmi *v1.VirtualMachineInstance, migrationUID string, timeout int) *v1.VirtualMachineInstance {
+		By("Waiting until the migration is completed")
+		Eventually(func() bool {
+			vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+			Expect(err).To(BeNil())
+
+			Expect(vmi.Status.MigrationState).ToNot(BeNil())
+
+			if vmi.Status.MigrationState.Completed {
```

Comment:
it seems like we'd want something in the MigrationState that indicates whether or not the cancellation was successful or not.